project.clj

```diff
@@ -20,6 +20,7 @@
                  [ring/ring-anti-forgery "1.2.0"]
                  [org.clojure/tools.logging "0.4.0"]
                  [camel-snake-kebab "0.4.0"]
+                 [enlive "1.1.6"]
                  [nilenso/honeysql-postgres "0.2.3"]]
   :profiles {:dev {:dependencies [[helpshift/faker "0.2.0"]]
                    :plugins [[com.jakemccrary/lein-test-refresh "0.22.0"]]}}
```

Comment:
It might be better to not have a `lein profile` in your `project.clj` and have it in `~/.lein/profiles.clj`

Reply:
@trycatcher We use the `faker` dependency to generate sample test data so we need the `:profiles` key here.

Reply:
@hargup I meant the `:plugins` bit. `lein-test-refresh` is a lein plugin and can be moved to your profiles.clj